src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/armor/SlimefunArmorTask.java

```diff
@@ -0,0 +1,82 @@
+package io.github.thebusybiscuit.slimefun4.implementation.tasks.armor;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+
+import io.github.thebusybiscuit.slimefun4.api.items.HashedArmorpiece;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem;
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile;
+import io.github.thebusybiscuit.slimefun4.core.attributes.Radioactive;
+import io.github.thebusybiscuit.slimefun4.implementation.Slimefun;
+import io.github.thebusybiscuit.slimefun4.implementation.items.armor.SlimefunArmorPiece;
+
+/**
+ * The {@link SlimefunArmorTask} is responsible for handling {@link PotionEffect PotionEffects} for
+ * {@link Radioactive} items or any {@link SlimefunArmorPiece}.
+ * It also handles the prevention of radiation through a Hazmat Suit
+ *
+ * @author TheBusyBiscuit
+ * @author martinbrom
+ * @author Semisol
+ *
```

Comment:
Does this still handle potion effects for radioactive items? I thought that was the Radioactive task, also there is an empty line at the end of the javadoc